test/winter_onboarding_2021/fleet_management/handlers/cab_test.clj

```diff
@@ -0,0 +1,21 @@
+(ns winter-onboarding-2021.fleet-management.handlers.cab-test
+  (:require [clojure.test :refer [deftest is testing use-fixtures]]
+            [winter-onboarding-2021.fleet-management-service.handlers.cab :as cab]
+            [winter-onboarding-2021.fleet-management.fixtures :as fixtures]))
+
+(use-fixtures :once fixtures/config fixtures/db-connection)
+(use-fixtures :each fixtures/clear-db)
+
+(deftest add-cab
+  (testing "POST  /cabs/ endpoint with valid cab data, shoudld redirect to '/cabs/new?success=true' "
+    (is {:status 302
```

Comment:
Need to update these tests. They must have a :flash key now

Reply:
Can you add seperate testing blocks to check that appropriate flash messages are set.

Reply:
Need more clarification:
There are two `testing` blocks which will check for a successful and unsuccessful addition of a new cab.

Should I break those two `testing` blocks into two `deftest` blocks?